react/package.json

```diff
@@ -11,8 +11,7 @@
     "url": "https://github.com/sparkdesignsystem/spark-design-system"
   },
   "dependencies": {
-    "@sparkdesignsystem/spark": "^11.0.0",
-    "@sparkdesignsystem/spark-react": "^1.0.1",
+    "@sparkdesignsystem/spark": "^12.0.0",
```

Comment:
shouldnt have this as a dep, since we're bringing in local files

Reply:
I was thinking for the cases where the local files bring it in as a dep. What do you think?